src/components/plots/FeaturesVsUMIsScatterplot.jsx

```diff
@@ -41,7 +47,10 @@ const FeaturesVsUMIsScatterplot = (props) => {
 
 FeaturesVsUMIsScatterplot.propTypes = {
   config: PropTypes.object.isRequired,
-  plotData: PropTypes.array,
+  plotData: PropTypes.oneOfType([
```

Comment:
Can you draw a `FeaturesVsUMIsScatterplot` without having `plotData`? Because looking at the usage and implementation of `FeaturesVsUMIsScatterplot`, it looks to me like this `plotData` property should be required.

Reply:
I just checked, it doesn't break but rather just doesn't get rendered. That being said, I think it's just lucky that it doesn't break. I considered added a test for `null` plotData but I think it makes more sense to just update the `defaultProps.plotData = []` which we are already testing for. Let me know if you'd prefer an alternative

Reply:
That sounds like a good solution